python/private/py_wheel.bzl

```diff
@@ -301,6 +305,9 @@ def _py_wheel_impl(ctx):
     if ctx.attr.summary:
         metadata_contents.append("Summary: %s" % ctx.attr.summary)
 
+    for label, url in sorted(ctx.attr.project_urls.items()):
```

Comment:
Please check for the max length of the label. The label is limited to 32 characters.

Reply:
Thank you for pointing out. A check has been added, and will fail out if the length is over limit. Corresponding test is also added.